src/stories/DataResultItem.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Meta } from '@storybook/react/types-6-0';
+import { Story } from '@storybook/react';
+import DataResultItem, { DataResultItemProps } from '../components/molecules/DataResultItem';
+
+export default {
+    title: 'DataExplorer/DataExplorer/Item',
+    component: DataResultItem,
+} as Meta;
+
+const Template: Story<DataResultItemProps> = (args) => <DataResultItem {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+    title: 'This is an title',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+    tags: ['Energi', 'Miljø', 'Helse', 'Pils', 'Raketter'],
+    visualisationType: 'thresholdchart',
+};
+
+Primary.storyName = 'Data explorer result item';
```

Comment:
Couldn't this just be called ```Item```, to keep the naming convention the same as the Dashboard. So you could edit line 7 to be ```title: 'DataExplorer/DataExplorer',```, and this to ```Item```.